torchgeo/datasets/airphen.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Airphen dataset."""
+
+from typing import Any, Optional
+
+import matplotlib.pyplot as plt
+from matplotlib.figure import Figure
+
+from .geo import RasterDataset
+from .utils import RGBBandsMissingError, percentile_normalization
+
+
+class Airphen(RasterDataset):
+    """Airphen dataset.
+
+    `Airphen <https://ideol.sakura.ne.jp/img/20170123_HiphenAirphenKeyfeatures.pdf>`__
+    is a multispectral scientific camera developed by agronomists and photonics
+    engineers at `Hiphen <https://www.hiphen-plant.com/>`_ to match plant measurements
+    needs and constraints. Its high flexibility, ease of use and radiometric quality
+    give you a wide range of opportunities to develop your customized applications.
+
+    Main characteristics:
+
+    * 6 Synchronized global shutter sensors
+    * Sensor resolution 1280 x 960 pixels
+    * Data format (.tiff, 12 bit)
+    * SD card storage
+    * Metadata information: Exif and XMP
+    * Internal or external GPS
+    * Synchronization with different sensors (TIR, RGB, others)
+
+    Key features:
+
+    * Customized sensing configurations
+    * Easy integration
+    * Full control and configuration
+    * Powerful analysis with a specific Agisoft module
+
+    If you use this dataset in your research, please cite the following paper:
```

Comment:
Reading this, I'm wondering if we should ask users to also cite torchgeo if they use this dataset in their paper?

Reply:
If someone uses TorchGeo to load this dataset, then they should cite TorchGeo. This is true for all datasets.

I agree that it's kind of weird to cite a paper that uses Airphen instead of a paper published by Hiphen themselves. I emailed Hiphen to ask how to cite their data and this is the reply I got:

> You contacted us to know how to cite our Airphen camera in your scientific papers. In general, “Airphen camera from Hiphen” is sufficient, but you could also cite the paper attached to this email as it is using intensively the Airphen camera.

"the paper attached" is the link I used.